frontend/src/utils/reused.js

```diff
@@ -1,3 +1,26 @@
 const BASE_URL = import.meta.env.DEV ? "http://localhost:3000" : ""; // official database url
+const LOGIN_SUCCESS = "password accepted";
 
-export { BASE_URL };
+const handleLogin = async (formData, responseSetter, attemptLogin) => {
```

Comment:
Why do you have two files, `utils.js` and `reused.js`?
I believe John mentioned this before, _utils_ are by definition something that is reused. So I'd put all these functions in the same file `utils.js`. If you have a lot and want to split, you can have multiple files, named, for example, `loginUtils.js`, `searchUtils.js` or whatever makes sense in your case.